packages/dashboard/src/components/DashboardFilters/DashboardFiltersEditor/components/FilterEditor.tsx

```diff
@@ -55,7 +55,11 @@ const FilterEditor: React.FC<FilterEditorProps> = ({
       if (!vizLookup) return false
       const viz = config.visualizations[vizKey] as Visualization
       if (viz.type === 'dashboardFilters') return false
-      const vizName = viz.general?.title || viz.title || vizKey
+      let vizName = viz.general?.title || viz.title || vizKey
+      if (viz.visualizationType === 'markup-include') {
+        vizName = viz.contentEditor.title || vizKey
+      }
+
       nameLookup[vizKey] = vizName

```

Comment:
I think you should remove these lines since they are duplicates of the returned value from getUsedByOptions. getUsedByOptions can be moved up in the file and called directly on ln 63.